### PR TITLE
add requested layout changes for view evaluation screen

### PR DESCRIPTION
**Pull request by emmaacha**
Ticket ID: [Create View Evaluation Administration Screen#40](https://github.com/nerubia/kh360-react/issues/40)

### CODE REVIEW

File: src/features/admin/evaluation-administrations/[id]/ViewEvaluationHeader.tsx [hunk 1]
```diff
@@ -61,6 +61,34 @@ export const ViewEvaluationHeader = () => {
                 Progress
               </LinkButton>
             ) : null}
+            <LinkButton
+              variant='primary'
+              size='medium'
```
Comment (jatnerubia) [inline src/features/admin/evaluation-administrations/[id]/ViewEvaluationHeader.tsx:66]: default is medium 🤔

File: src/redux/slices/evaluationAdministrationSlice.ts [hunk 2]
```diff
@@ -61,6 +61,22 @@ export const generateStatusEvaluationAdministration = createAsyncThunk(
   }
 )
 
+export const deleteEvaluationAdminsitration = createAsyncThunk(
```
Comment (jatnerubia) [inline src/redux/slices/evaluationAdministrationSlice.ts:64]: typo 🤔

**Review by jatnerubia — CHANGES_REQUESTED** [review]
just a few changes ^^

**Review by Dev-Nino — APPROVED** [review]

**Review by jatnerubia — APPROVED** [review]
Nice

**Pull request merged**